src/tasks/task-capture-point-inequality.cpp

```diff
@@ -0,0 +1,138 @@
+//
+// Copyright (c) 2017 CNRS, NYU, MPI Tübingen
```

Comment:
You can change the year and add PAL Robotics here

Reply:
done.